Meetups/Readme.md

```diff
@@ -0,0 +1,54 @@
+# Meetup Resources
+
+In this folder, you can find the presentation and minutes of each meetup.
+This readme contains the links to minutes, presentation files and the recordings of the presentations.
+
+Note: Presentation files use the pattern `YEAR-MONTH-WoTCG-EVENTNAME-PRESENTERNAME.FILETYPE`.
+
+## Meetup 18
```

Comment:
@relu91 let me know what you think about the readme structure. If you give thumbs up, I will do this to the other meetups